Controllers/Subject.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace University_advisor
+{
+    public class Subject
+    {
+        private string name;
+        private double rating;
+
+        public Subject(string name, double rating)
+        {
+            this.name = name;
+            this.rating = rating;
+        }
+
+        public string Name { get { return name; } set { name = value; } }
```

Comment:
if this is only POCO class (has only fields, no logic in validations or nothing), it can in theory be defined as:
public class Subject
{
public string Name {get;set;}
public double Rating {get;set;}
}

and it would do exactly same.


Or leave ctor (constructor shortening) and have it with my mentioned properties. As currently as it's written, it's not needed